rpmio/rpmlog.c

```diff
@@ -237,6 +237,89 @@ const char * rpmlogLevelPrefix(rpmlogLvl pri)
     return prefix;
 }
 
+static const char * rpmlogLevelColor(rpmlogLvl pri)
+{
+    return rpmlogMsgPrefixColor[pri&0x7];
+}
+
+static int rpmlogDefault(FILE *stdlog, rpmlogRec rec)
+{
+    static const char fubar[] =
+	"Error occurred during writing of a log message";
+    FILE *msgout = (stdlog ? stdlog : stderr);
+    const char * colorOn = isatty(fileno(msgout))
```

Comment:
I only commited as-is to a separate branch to make PR from without testing as a favour to @n3npq, unless he attends to, I'll probably get to it eventually.. ;)